demos/tcp/aws_tcp_echo_client_single_task.c

```diff
@@ -323,7 +323,7 @@ static void prvEchoClientTask( void * pvParameters )
                 /* If an error occurred it will be latched in xReceivedBytes,
```

Comment:
Apparently, the test and demo projects do not use the TCP sliding windows ( `ipconfigUSE_TCP_WIN` ). Which is a pity, because it would make them run considerably faster.

I will repair it, and re-test it in real hardware, both with TCP sliding enabled and disabled.
( see my commit: "xWinProps undefined when ipconfigUSE_TCP_WIN equals 1" )

Reply:
I re-tested the code, both with and without sliding windows. I had an echo server that can trigger several errors, like:
- Corrupted data
- Bad length ( less than expected )
- Sudden disconnection

That all worked well. Corrupted data leads to an ASSERT.